src/components/Fundraiser.tsx

```diff
@@ -641,7 +640,7 @@ export const FundraiserPost: CustomPostType = {
           width,
           height,
           staticData.nonprofitInfo,
-          staticData.coverImageUrl,
+          coverImageUrl,
```

Comment:
Doesn't this need to be kept in a state variable?

Reply:
why would we keep a potential 404 image link in a state variable?

Reply:
The idea would be to put the coverImageUrl check in the initializer function for the state, so that the check is only called on the first render, serverside, when the component mounts. If the component rerenders in a user session, it couldn't need to be checked again